avro/service-preferences.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name":"servicePreferences",
+  "type":"record",
+  "namespace": "dto",
+  "doc":"Schema for cosmos api container 'services-preferences'",
+  "fields":[
+    {
+      "name":"id",
+      "type":"string",
+      "default":"undefined"
+    },
+    {
+      "name":"fiscalCode",
```

Comment:
Since it won't be a fiscal code, I'd rather use something like `userId` or `userPDVId`

Reply:
renamed in https://github.com/pagopa/io-functions-elt/pull/80/commits/25a1ac5b028f7c6d293a9c9430bfb0aa41968647